include/CL/opencl.hpp

```diff
@@ -8199,6 +8370,24 @@ class CommandQueue : public detail::Wrapper<cl_command_queue>
         return result;
     }
 
+     inline void* enqueueMapImage(
+         const Image& buffer,
```

Comment:
Nitpick: Since this is an "Image" argument I think it's a little confusing to call it a "buffer".

```suggestion
         const Image& image,
```

(We'll need to change where it's used below, also.)

Reply:
Good point! Done.